indico/modules/events/persons/controllers.py

```diff
@@ -152,9 +187,11 @@ def _process(self):
             if not persons[principal.email].get('no_account'):
                 persons[principal.email]['roles']['no_account'] = True
                 num_no_account += 1
-
-        return WPManagePersons.render_template('management/person_list.html', self.event,
-                                               persons=person_list, num_no_account=num_no_account)
+        custom_roles = {'custom_{}'.format(r.id): {'name': r.name, 'code': r.code, 'color': r.color}
+                        for r in EventRole.query.with_parent(self.event).all()}
```

Comment:
why not `self.event.roles` (or whatever I called the backref of the relationship)?